http/src/main/resources/reference.conf

```diff
@@ -1171,6 +1171,11 @@ pubsub {
       interval = 3 seconds
       interruptAfter = 10 minutes
     }
+    appLivelinessCheck {
```

Comment:
Hmm... my assumption is that in most cases this will return immediately, and if we are retrying we don't actually expect it to ever be available, so it doesn't matter much what this is configured to in my head